examples/step-70/step-70.cc

```diff
@@ -532,68 +567,86 @@ namespace Step70
     LA::MPI::BlockVector       locally_relevant_solution;
     LA::MPI::BlockVector       system_rhs;
 
+    // Let us move to the particles side of this program. There are two
+    // Particles::ParticleHandler objects used to couple the solid with the
+    // fluid, and to describe the passive tracers. These, in many ways, play a
+    // role similar to the DoFHandler class used in the discretization, i.e.,
+    // they provide for an enumeration of particles and allow querying
+    // information about each particle.
+    Particles::ParticleHandler<spacedim> tracer_particle_handler;
+    Particles::ParticleHandler<spacedim> solid_particle_handler;
+
     // For every tracer particle, we need to compute the velocity field in its
     // current position, and update its position using a discrete time stepping
     // scheme. We do this using distributed linear algebra objects, where the
     // owner of a particle is set to be equal to the process that generated that
-    // particle at time t=0. This information is stored for every process in the
-    // `owned_tracer_particles` IndexSet, that indicates which particles the
+    // particle at time $t=0$. This information is stored for every process in
+    // the `owned_tracer_particles` IndexSet, that indicates which particles the
     // current process owns.
     //
     // Once the particles have been distributed around to match the process that
     // owns the region where the particle lives, we will need read access from
     // that process on the corresponding velocity field. We achieve this by
-    // filling a read only velocity vector field, that contains the relevant
+    // filling a read only velocity vector field that contains the relevant
     // information in ghost entries. This is achieved using the
     // `relevant_tracer_particles` IndexSet, that keeps track of how things
     // change during the simulation, i.e., it keeps track of where particles
-    // that I own have ended up being, and who owns the particles that ended up
-    // in my subdomain.
+    // that the current process owns have ended up being, and who owns the
+    // particles that ended up in my subdomain.
     //
     // While this is not the most efficient strategy, we keep it this way to
-    // illustrate how things would work in a real FSI problem. If a particle
-    // is linked to a specific solid degree of freedom, we are not free to
-    // choose who owns it, and we have to communicate this information around.
-    // We illustrate this here, and show that the communication pattern is
-    // point-to-point, and negligible in terms of total cost of the algorithm.
+    // illustrate how things would work in a real fluid-structure
+    // intraction(FSI_ problem. If a particle is linked to a specific solid
```

Comment:
```suggestion
    // intraction (FSI) problem. If a particle is linked to a specific solid
```

Reply:
Only this minor typo. For the rest: thanks  @bangerth for taking the time to do this!

:)

Reply:
Wait, shouldn't this be "interaction" rather than "intraction"?

Reply:
Yes!

Reply:
Yes, I had to laugh as well :-)